dokku-server.yml

```diff
@@ -12,6 +12,26 @@
       tags:
         - dokku-ssh-keys
 
+    - name: Make sure /var/log/dokku directory exists
+      file:
+        path: /var/log/dokku
+        owner: syslog
+        group: dokku
+        mode: 0775
+        state: directory
+
+    - name: Make sure that dokku log files exist with correct ownership
+      file:
+        path: "/var/log/dokku/{{ item }}"
+        owner: syslog
+        group: dokku
+        mode: 0664
+        state: touch
+      with_items:
+        - letsencrypt.log
+        - events.log
+        - dokku-daemon.log
```

Comment:
I'd prefer we don't involve other logs in this change - yes, they might need the same treatment, but since this is a temporary workaround, I don't want us to go messing with things where we might not understand the consequences

Reply:
Well, we do understand that if dokku-daemon.log is owned by root:root, we'll get similar ownership change error. Just a word of warning.

Logrotate is going to attempt, as syslog:dokku to try and chown a gz file to that of the original file (in this case owned by root:root) and it will fail.

Reply:
I didn't want to make changes for something where we haven't seen the problem before, and which could have unforeseen consequences.

So is the reason we haven't seen this before, that

- with letsencrypt.log.1 failing, it didn't continue to dokku-daemon
- when you tested changing ownership, you changed it for both all files?

Reply:
confirmed. this is needed. Thanks @dnk8n